variants/backend-base/config/secrets.yml

```diff
@@ -0,0 +1,25 @@
+# Do NOT put secrets directly into this file. All secrets should be loaded from ENV!
+# Be sure to restart your server when you modify this file.
+
+default: &default
+  # Your secret key is used for verifying the integrity of signed cookies.
+  # If you change this key, all old signed cookies will become invalid!
+  # Make sure the secret is at least 30 characters and all random,
+  # no regular words or you'll be exposed to dictionary attacks.
+  # You can use `rails secret` to generate a secure secret key.
+  secret_key_base: "<%= ENV['RAILS_SECRET_KEY_BASE'] %>"
+  active_record_encryption_primary_key: "<%= ENV['ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY'] %>"
+  active_record_encryption_deterministic_key: "<%= ENV['ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY'] %>"
+  active_record_encryption_key_derivation_salt: "<%= ENV['ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT'] %>"
```

Comment:
If you nest keys here then you have to use hash key syntax to get the value e.g.

```yaml
aaa:
  bb: "foo"

# Rails.application.secrets.aaa.bb # breaks
# Rails.application.secrets.aaa[:bb] # works
```
This felt a bit ugly so I flattened the secrets instead